### PR TITLE
Renamed line item property from Pickup Store to Store Pickup(#2nvxwh1)

### PR DESCRIPTION
**Pull request by disha1202**

### CODE REVIEW

File: src/views/OrderDetail.vue [hunk 1]
```diff
@@ -129,11 +129,11 @@ export default defineComponent({
     },
     markBopisItem (item: any, event: any) {
       if(this.isBopisItem(item)){
-        item.properties = item.properties.filter((property: any) => !(property.name === '_pickupstore' || property.name === 'Pickup Store'))
+        item.properties = item.properties.filter((property: any) => !(property.name === '_pickupstore' || property.name === 'Store Pickup' || property.name === 'Pickup Store'))
```
Comment (ymaheshwari1) [inline src/views/OrderDetail.vue:132]: Add a comment that added the check `Pickup Sotre` for backward compatibility.

**Review by ymaheshwari1 — CHANGES_REQUESTED** [review]

File: src/views/OrderDetail.vue (hunk 1 above)
Comment (disha1202) [reply src/views/OrderDetail.vue:133]: Added comment.

File: src/views/OrderDetail.vue [hunk 2]
```diff
@@ -129,6 +129,7 @@ export default defineComponent({
     },
     markBopisItem (item: any, event: any) {
       if(this.isBopisItem(item)){
+        // Kept Pickup Store check for orders that have been already marked BOPIS. Will remove after sometime.
```
Comment (ymaheshwari1) [inline src/views/OrderDetail.vue:132]: Update the comment to `Need to remove the `Pickup Store` check, currently kept it for backward compatibility.

**Review by ymaheshwari1 — CHANGES_REQUESTED** [review]

File: src/views/OrderDetail.vue (hunk 2 above)
Comment (disha1202) [reply src/views/OrderDetail.vue]: Updated comment

**Review by ymaheshwari1 — APPROVED** [review]

**Review by adityasharma7 — APPROVED** [review]

**Pull request merged**